plugins/modules/packaging/os/rpm_ostree_pkg.py

```diff
@@ -125,6 +140,10 @@ def ensure(self):
         for pkg in self.params['name']:
             cmd.append(pkg)
             results['packages'].append(pkg)
+            
+        # Append apply-live
+        if self.apply_live and self.state not in ('absent'):
```

Comment:
Otherwise you are checking whether `self.state` is in `('a', 'b', 's', 'e', 'n', 't')` :-)
```suggestion
        if self.apply_live and self.state not in ('absent', ):
```